drafts/bitcartel-nu0-network-handshake/draft.rst

```diff
@@ -0,0 +1,222 @@
+::
+
+  ZIP: ???
+  Title: Network Upgrade Zero ("OverWinter") Network Handshake
+  Author: Simon Liu <simon@z.cash>
+  Comments-Summary: No comments yet.
+  Category: Process
+  Created: 2018-01-15
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be interpreted as described in RFC 2119.
+
+"Legacy" - pre-NU0
+
+"NU0" - Network upgade zero
+
+"Overwinter" - Code-name for NU0
+
+Abstract
+========
+
+This proposal defines an upgrade to network handshake format required for Network Upgrade Activation Mechanism [#zip-0???]_.
+
+Related to [#zip-0143]_
```

Comment:
Add references to the other Overwinter ZIPs (202, 203), with brief descriptions of what they do (similarly for 143), so that this is acting as the meta-ZIP for Overwinter deployment.

(It would be disruptive to add a separate meta-ZIP now.)